projects/components/src/overlay/overlay.ts

```diff
@@ -1,7 +1,12 @@
 import { TemplateRef, Type } from '@angular/core';
+import { Observable } from 'rxjs';
 
 export interface OverlayConfig {
   showHeader?: boolean;
   title?: string;
   content: TemplateRef<unknown> | Type<unknown>;
 }
+
+export interface ConfirmationModalConfig extends OverlayConfig {
```

Comment:
Confirmation is just another modal in the library, it shouldn't be part of the overlay api itself.

Reply:
+1